project/script_IR.tcl

```diff
@@ -1,20 +1,31 @@
-############################################################
-## This file is generated automatically by Vivado HLS.
-## Please DO NOT edit it.
-## Copyright (C) 1986-2017 Xilinx, Inc. All Rights Reserved.
-############################################################
+# Script to generate project for IR
+#   vivado_hls -f script_IR.tcl
+#   vivado_hls -p inputrouter
+# WARNING: this will wipe out the original project by the same name
+
+# create new project (deleting any existing one of same name)
 open_project -reset inputrouter
+
+# source files
+set CFLAGS {-std=c++11 -I../TrackletAlgorithm}
 set_top InputRouter
-add_files ../TrackletAlgorithm/InputRouter.cpp -cflags "-std=c++11"
-add_files -tb ../TestBenches/InputRouter_test.cpp -cflags "-I../TrackletAlgorithm -std=c++11"
-add_files -tb ../emData/IL
-open_solution -reset "solution1"
+add_files ../TrackletAlgorithm/InputRouter.cpp -cflags "$CFLAGS"
+add_files -tb ../TestBenches/InputRouter_test.cpp -cflags "CFLAGS"
+
+# data files
+add_files -tb ../emData/IL/
+
+# solutions
+open_solution "solution1"
+source settings_hls.tcl
 source set_fpga.tcl
-create_clock -period 4 -name default
-#source "./projrouter/solution1/directives.tcl"
-csim_design -compiler gcc
+create_clock -period 250MHz -name default
+
+csim_design -compiler gcc -mflags "-j8"
```

Comment:
is -j8 passed to make or to the compiler? If the latter then maybe it should be set to something like the output of the `nproc` command.